portsyncd/portsyncd.cpp

```diff
@@ -85,7 +86,12 @@ int main(int argc, char **argv)
         netlink.registerGroup(RTNLGRP_LINK);
         cout << "Listen to link messages..." << endl;
 
-        handlePortConfigFile(p, port_config_file);
+        if (port_config_file.empty())
```

Comment:
true if the string length is 0, false otherwise.

Reply:
my bad. fixed